public_html/wp-content/plugins/wordcamp-payments/includes/payment-request.php

```diff
@@ -1130,7 +1130,7 @@ public static function _generate_payment_report_jpm_checks( $args ) {
 
 			$payable_to = WCP_Encryption::maybe_decrypt( get_post_meta( $post->ID, '_camppayments_payable_to', true ) );
 			$payable_to = html_entity_decode( $payable_to ); // J&amp;J to J&J
-			$countries = WordCamp_Budgets::get_valid_countries_iso3166();
+			$countries = wcorg_get_countries( array( 'include_alpha3' => true ) );
 			$vendor_country_code = get_post_meta( $post->ID, '_camppayments_vendor_country_iso3166', true );
```

Comment:
There are a bunch of these meta field names that reference 3166. I guess they're _technically_ still accurate, since CLDR uses 3166 for the alpha-2/3 code; is that right?

It might still be good to have a comment somewhere explaining that they're just a legacy name, though, and that CLDR should be considered authoritative throughout the codebase now?

Reply:
Good idea. Addressed in d933d594